big_tests/tests/rest_client_SUITE.erl

```diff
@@ -127,6 +145,15 @@ init_per_testcase(TC, Config) ->
                    ],
     rest_helper:maybe_skip_mam_test_cases(TC, MAMTestCases, Config).
 
+end_per_testcase(config_can_be_changed_by_all = CaseName, Config) ->
+    Host = ct:get_config({hosts, mim, domain}),
+    MUCLightHost = <<"muclight.", Host/binary>>,
+    dynamic_modules:restart(Host, mod_muc_light,
```

Comment:
A less error-prone way would be to call `dynamic_modules:save_modules/2` before the testcase and `dynamic_modules:restore_modules/2` after. See amp_big_SUITE.erl for example